src/main/java/com/rw/legion/columncheck/BooleanChecker.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2017 Republic Wireless
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rw.legion.columncheck;
+
+import com.google.gson.JsonObject;
+
+public class BooleanChecker extends BoolChecker {
+
+    public BooleanChecker(JsonObject json) {
```

Comment:
I'm not sure I'm following the need for this? How does it differ from `BoolChecker`?

Reply:
It's the same. Just the actual full name, rather than `Bool`. 

`Bool` doesn't fit with `Integer`, `Float`, `String`. `Boolean` does. But `Bool would need to remain for backwards compatibility.

Reply:
Ah, I see. If we're just going for a name change with backwards compatibility, would it make more sense to rename the main class BooleanChecker, make BoolChecker extend BooleanChecker, and then add a comment that it's there for backwards compatibility?

Reply:
True. I'll make that update